cli/src/commands/logs.ts

```diff
@@ -15,29 +16,29 @@ export default class Logs extends Command {
   public static description = 'show logs';
 
   public static examples = [
-`// retrieve all logs (except "function invocation")
-$ ${Command.cliBinName} logs
-`,
-`// retrieve all logs (including "function invocation")
-$ ${Command.cliBinName} logs --all
-`,
-`// tail all logs
-$ ${Command.cliBinName} logs --follow
-`,
-`// retrieve logs of specific app
-$ ${Command.cliBinName} logs cool-dragon-17
-`,
-`// ISO
-$ ${Command.cliBinName} logs --since 2018-03-09T22:12:21.861Z
-`,
-`// offset format
-$ ${Command.cliBinName} logs --since 3d
-$ ${Command.cliBinName} logs --since 13hours
-$ ${Command.cliBinName} logs --since 9s
-`,
-`// show all logs from 2 minutes ago and follow in real time
-$ ${Command.cliBinName} logs --since 2m --follow`,
-];
+    dedent`// retrieve all logs (except "function invocation")
+    $ ${Command.cliBinName} logs`,
```

Comment:
Really don't like this indentation, I'd rather indent this line further in.  (I'm guessing it will work, too!)